docs/integrations/amazon-aws/amazon-opensearch-service.md

```diff
@@ -8,20 +8,345 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+      "volumeType": "gp2"
+    },
+    "accessPolicies": {...},
+  "responseElements": {
+    "domainStatus": {
+      "created": true,
+      "clusterConfig": {
+        "zoneAwarenessEnabled": false,
+        "instanceType": "m4.large.search",
+        "dedicatedMasterEnabled": false,
+        "instanceCount": 1
+      },
+      "cognitoOptions": {
+        "enabled": false
+      },
+      "encryptionAtRestOptions": {
+        "enabled": false
+      },
+      "advancedOptions": {
+        "rest.action.multi.allow_explicit_index": "true"
+      },
+      "upgradeProcessing": false,
+      "snapshotOptions": {
+        "automatedSnapshotStartHour": 0
+      },
+      "eBSOptions": {
+        "eBSEnabled": true,
+        "volumeSize": 10,
+        "volumeType": "gp2"
+      },
+      "engineVersion": "OpenSearch_1.0",
+      "processing": true,
+      "aRN": "arn:aws:es:us-west-1:123456789012:domain/test-domain",
+      "domainId": "123456789012/test-domain",
+      "deleted": false,
+      "domainName": "test-domain",
+      "accessPolicies": {...},
+  "requestID": "12345678-1234-1234-1234-987654321098",
+  "eventID": "87654321-4321-4321-4321-987654321098",
+  "eventType": "AwsApiCall",
+  "recipientAccountId": "123456789012"
+}
+
+```
+
+### **Sample queries**
+
+Successful Events by Event Name
+
+```
+
+account={{account}} region={{region}} namespace=aws/es "\"eventsource\":\"es.amazonaws.com\""
+| json "userIdentity", "eventSource", "eventName", "awsRegion", "sourceIPAddress", "userAgent", "eventType", "recipientAccountId", "requestParameters", "responseElements", "requestID", "errorCode", "errorMessage" as userIdentity, event_source, event_name, region, src_ip, user_agent, event_type, recipient_account_id, requestParameters, responseElements, request_id, error_code, error_message nodrop
+| where event_source = "es.amazonaws.com" 
+| json field=userIdentity "accountId", "type", "arn", "userName"  as accountid, type, arn, username nodrop
+| parse field=arn ":assumed-role/*" as user nodrop 
+| parse field=arn "arn:aws:iam::*:*" as accountid, user nodrop
+| json field=requestParameters "domainName" as domainname nodrop
+| if (isBlank(accountid), recipient_account_id, accountid) as accountid
+| where (tolowercase(domainname) matches tolowercase("{{domainname}}")) or isBlank(domainname)
+| if (isEmpty(error_code), "Success", "Failure") as event_status
+| if (isEmpty(username), user, username) as user
+| count as event_count by event_name
+| sort by event_count, event_name asc
+```
+
+Write Latency by Domain Name (Metrics-based)
+
+```
+
+account={{account}} region={{region}} namespace=aws/es domainname={{domainname}} !nodeid=* metric=WriteLatency statistic = average | avg by domainname 
+```
+
+## **Collecting logs and metrics for the Amazon OpenSearch app**
+
+### **Collecting Metrics for Amazon OpenSearch**
+
+1. Configure a [Hosted Collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/).  
```

Comment:
Can you please the steps from this document: https://help.sumologic.com/docs/integrations/amazon-aws/dynamodb/#collect-metrics-for-amazon-dynamodb